src/app/features/home/capture-tab/capture-details/options-menu/options-menu.component.html

```diff
@@ -1,4 +1,7 @@
 <mat-nav-list *transloco="let t">
+  <a mat-list-item (click)="openLink(options.Share)">
```

Comment:
It might be a good idea to hide the share option if `diaBackendAssetID` is `undefined`, so the user won't be able to access the share button if the Capture can't be shared.